app/blog/posts/good-python-code.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: "Everything I've learnt about writing good Python code"
+publishedAt: "2024-04-17"
+description: Speedrun your way to becoming a good python developer and don't make the same mistakes I did
+---
+
+I've always used python for my interviews but it wasn't until about 6 months ago that I started to use python for all sorts of projects - from simple machine learning scripts to full blown applications. In the proccess, I've learnt a lot about what constitutes good python code and I hope that you'll find the tips in this article useful in your own projects.
+
+I've been lucky enough to have Jason (@jxnlco on twitter) review a good chunk of my code and I've found that these few things have made a massive difference in my code quality.
+
+1. using the `@classmethod` decorator
+2. taking time to learn about common libraries
+3. writing simpler functions
+4. being a bit lazier - earn the abstraction
+5. decouple your implementation
+
+## Use the classmethod decorator
```

Comment:
you cna call out the fact that instructor's new api looks like this 

from_opena
from_anthropic